app/routes.js

```diff
@@ -1,4 +1,8 @@
 const users = require('./controllers/users.js'),
-  signUpValidator = require('./middlewares/validator').signUpValidator;
+  signUpValidator = require('./middlewares/validator').signUpValidator,
+  signInValidator = require('./middlewares/validator').signInValidator;
 
-exports.init = app => app.post('/users', [signUpValidator], users.create);
+exports.init = app => {
+  app.post('/users', [signUpValidator], users.create),
+    app.post('/users/sessions', [signInValidator], users.signIn);
```

Comment:
This space is weird

Reply:
This space is being applied by the linter of the bootstrap. It is automatic.

Reply:
Yes, maybe. We create our linter in this moment. :+1: